kubernetes/helm_charts/core/nginx-private-ingress/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: private-ingress
+  namespace: {{ .Values.namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: private-ingress
+  template:
+    metadata:
+      labels:
+        app: private-ingress
+    spec:
+      containers:
+      - name: name
+        image: nginx
+        resources:
+          requests:
+            cpu: 100m
```

Comment:
Add readiness and liveliness probes - Check if nginx has a endpoint which gives 200